ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.11__amd64-nightly-4.11-upgrade-from-stable-4.10-from-stable-4.9.yaml

```diff
@@ -1,93 +0,0 @@
-base_images:
-  dev-scripts:
-    name: test
-    namespace: ocp-kni
-    tag: dev-scripts
-  openstack-installer:
-    name: "4.9"
-    namespace: ocp
-    tag: openstack-installer
-  upi-installer:
-    name: "4.9"
-    namespace: ocp
-    tag: upi-installer
-releases:
-  intermediate:
-    prerelease:
-      product: ocp
-      version_bounds:
-        lower: 4.10.0
-        upper: 4.11.0-0
-  latest:
-    prerelease:
-      product: ocp
-      version_bounds:
-        lower: 4.9.0
-        upper: 4.10.0-0
-  target:
-    candidate:
-      architecture: amd64
-      product: ocp
-      stream: nightly
-      version: "4.11"
-resources:
-  '*':
-    requests:
-      cpu: 100m
-      memory: 200Mi
-tests:
-- as: aws-ipi-f28
-  cron: 27 13 13 * *
-  steps:
-    cluster_profile: aws-qe
-    env:
-      BASE_DOMAIN: qe.devcluster.openshift.com
-      UPGRADE_CHECK_RUN_TAGS: '@upgrade-check and @4.9 and @aws-ipi and not @fips'
-      UPGRADE_PRE_RUN_TAGS: '@upgrade-prepare and @4.9 and @aws-ipi and not @fips'
-    test:
-    - as: set-upgrade-releases
-      commands: echo "${RELEASE_IMAGE_INTERMEDIATE},${RELEASE_IMAGE_TARGET}" > ${SHARED_DIR}/upgrade-edge
-      dependencies:
-      - env: RELEASE_IMAGE_INTERMEDIATE
-        name: release:intermediate
-      - env: RELEASE_IMAGE_TARGET
-        name: release:target
-      from: cli
-      resources:
-        requests:
-          cpu: 100m
-          memory: 200Mi
-    - chain: cucushift-upgrade
```

Comment:
Shall we also remove the chain file together?

Reply:
I think that can be done in a separate PR with extra check.

Reply:
I did some local check, the `cucushift-upgrade` chain is only referenced by this ci yaml file.

Reply:
Done